mon-pix/app/components/challenge-item-qrocm.js

```diff
@@ -9,8 +9,11 @@ import jsyaml from 'js-yaml';
 class ChallengeItemQrocm extends ChallengeItemGeneric {
   _hasError() {
     const allAnswers = this._getRawAnswerValue(); // ex. {"logiciel1":"word", "logiciel2":"excel", "logiciel3":""}
-    const hasAtLeastOneAnswer = _(allAnswers).hasSomeTruthyProps();
-    return _.isFalsy(hasAtLeastOneAnswer);
+    return this._hasEmptyAnswerFields(allAnswers);
+  }
+
+  _hasEmptyAnswerFields(answers) {
+    return _.filter(answers, (answer) => _.isEmpty(answer)).length;
```

Comment:
Une petite suggestion de simplification de la syntaxe 😄 

```suggestion
    return _.filter(answers, _.isEmpty).length;
```

Reply:
ah oui tellement ! 🤦‍♂️